main-packages-list.ocp

```diff
@@ -62,9 +62,12 @@ python3-pecan
 python3-pint >= 0.10.1-3.el9
 python3-proliantutils >= 2.16.0-0.20231026140721.5839129.el9
 python3-psutil
+python3-pyasn1 >= 0.5.1-3.el9
 python3-pycdlib
 python3-pyghmi >= 1.5.14-2.1.el9ost
+python3-pysnmp-lextudio >= 5.0.26-2.el9
 python3-scciclient >= 0.12.3-0.20230308201513.0940a71.el9
+python3-smi-lextudio >= 1.1.13-0.1.el9
```

Comment:
CI is failing because python3-smi appears to be installed in the container we're pulling from before we get there. I'm trying to track down why.

The python3-smi-lextudio package intentionally conflicts with python3-smi but (also intentionally) does not obsolete it since they are divergent packages. We can either remove python3-smi or we can Obsolete it, with the latter being a little bit dangerous.

Reply:
python-scciclient-0.15.0-0.20240124023324.a0d98f9.el9 has the correct dependencies - I think that might resolve the issue.